packages/text-field/Input.js

```diff
@@ -117,7 +117,12 @@ export default class Input extends React.Component {
   }
 
   isBadInput = () => this.inputElement.current.validity.badInput;
-  isValid = () => this.inputElement.current.validity.valid;
+  isValid = () => {
+    if (typeof this.props.isValid === 'boolean') {
```

Comment:
We're just checking to see if this prop exists, right?
```
if (this.props.isValid !== undefined)
```

Reply:
On second thought I'm not sure which way of putting it is less awkward, up to you

Reply:
I did `typeof this.props.isValid === 'boolean'`, because we want it to be `true` or `false`. If its anything else (null, undefined, 12, 'abc', etc.) then it shouldn't be considered.

Reply:
actually...I guess they would get a warning via PropTypes. I will change to your suggestion.